tools/dfx-orbit/src/args/review/list.rs

```diff
@@ -1,31 +1,43 @@
 //! CLI arguments for `dfx-orbit review list`.
 
 use clap::Parser;
-use orbit_station_api::ListRequestsInput;
+use orbit_station_api::{ListRequestsInput, SortDirection};
+
+use super::external_canister_operations;
+
+// TODO: Ideas what we could filter by:
+// - Filter by status: -> Only the ones which are in Created
+// - Filter by times -> There are four times that could be set
+// - Filter by request ids
 
 /// Reviews the next request.
 #[derive(Debug, Parser)]
 pub struct ReviewListArgs {
+    /// Show all request types, not only the ones related to canister management
+    #[clap(short, long)]
+    pub all: bool,
+
     /// Show only approvable requests.
     #[clap(short, long)]
     pub only_approvable: bool,
 }
 
 impl From<ReviewListArgs> for ListRequestsInput {
     fn from(args: ReviewListArgs) -> Self {
-        let ReviewListArgs { only_approvable } = args;
         Self {
             requester_ids: None,
             approver_ids: None,
             statuses: None,
-            operation_types: None,
+            operation_types: (!args.all).then(external_canister_operations),
```

Comment:
Is there a reason why canister management is set as a default?

could we have those as the args of `ReviewListArgs`, maybe something like:

```
#[derive(Debug, Parser)]
pub struct ReviewListArgs {
    /// Filter the list by the request operation types provided.
    #[clap(short, long)]
    pub requests: Option<Vec<String>>,
    
    /// Filter the list by the request groups provided.
    /// 
    /// e.g. `canister_management` group includes `fund`, `call`, `create` and `configure` 
    /// external canister operations
    #[clap(short, long)]
    pub groups: Option<Vec<String>>,
    
   /// Filter only by `canister_management` operations.
   #[clap(short, long, conflicts_with = "groups,requests")]
   pub only_canister_management: Option<bool>, 
}
```

and for the requests and groups provide a command that would give the list of options to the user.

Reply:
The reason that I think it makes sense to filter by `external_canisters` by default is that `dfx-orbit` is a tool to manage external canisters, at least at the moment. Other requests, such as managing  the users and groups is easier through the Web UI.

Therefore, I would expect that users would mostly care about external canisters at the moment. In the future, we might want to add other filters, similar to what you are proposing.